docs/store/datastore.md

```diff
@@ -167,6 +167,81 @@ redis_profile = project.get_datastore_profile("my_profile")
+
+Authentication against Azure services using a Service Principal:
+- `client_id` &mdash; This variable holds the client ID associated with an Azure Active Directory (AAD) application,
+which represents the Service Principal. In Azure, a Service Principal is used for non-interactive authentication, allowing applications to access Azure resources.
+The equivalent to this parameter in environment authentication is "AZURE_STORAGE_CLIENT_ID".
+- `client_secret` &mdash; This variable stores the client secret associated with the Azure AD application (Service Principal).
+The client secret is a credential that proves the identity of the application when it requests access to Azure resources.
+This key is sensitive information and is kept confidential.
+The equivalent to this parameter in environment authentication is "AZURE_STORAGE_CLIENT_SECRET".
+- `tenant_id` &mdash; This variable holds the Azure AD tenant ID, which uniquely identifies the organization or directory in Azure Active Directory.
+The equivalent to this parameter in environment authentication is "AZURE_STORAGE_TENANT_ID".
+credential authentication:
+- `credential` &mdash; TokenCredential or SAS token. The credentials with which to authenticate.
+This variable is sensitive information and is kept confidential.
+
+### DBFS data store profile
+
+```
+profile = DatastoreProfileDBFS(name="test_profile", endpoint_url="abc-d1e2345f-a6b2.cloud.databricks.com", token=token)
+ParquetTarget(path="ds://test_profile/path/to/parquet.pq")
+```
+
+`DatastoreProfileDBFS` init parameters:
+- `name` &mdash; Name of the profile.
+- `endpoint_url` &mdash; A string representing the endpoint URL of the DBFS service.
+The equivalent to this parameter in environment authentication is "DATABRICKS_HOST".
+- `token` &mdash; A string representing the secret key used for authentication to the DBFS service. 
+For privacy reasons, it's tagged as a private attribute, and its default value is `None`.
+The equivalent to this parameter in environment authentication is "DATABRICKS_TOKEN".
+
+### GCS data store profile
+```
+profile = DatastoreProfileGCS(name="test_profile",credentials_path="/local_path/to/gcs_credentials.json")
+ParquetTarget(path="ds://test_profile/gcs_bucket/path/to/parquet.pq")
+```
+
+`DatastoreProfileGCS` init parameters:
+- `name` &mdash; Name of the profile.
+- `credentials_path` &mdash; A string representing the local JSON file path that contains the authentication parameters required by the GCS API.
+The equivalent to this parameter in environment authentication is "GOOGLE_APPLICATION_CREDENTIALS."
+- `gcp_credentials` &mdash; A JSON in a string format representing the authentication parameters required by GCS API. 
+For privacy reasons, it's tagged as a private attribute, and its default value is `None`.
+The equivalent to this parameter in environment authentication is "GCP_CREDENTIALS".
+The code prioritizes `credentials_path` over `gcp_credentials` if both are not None.
```

Comment:
We just changed that to:

The code prioritizes `gcp_credentials` over `credentials_path` (without "if both are not None" )